doc/en/goodpractices.rst

```diff
@@ -7,12 +7,12 @@ Good Integration Practices
 Install package with pip
 -------------------------------------------------
 
-For development, we recommend to use virtualenv_ environments and pip_
+For development, we recommend you use venv_ for virtual environments and pip_
```

Comment:
Good suggestion, but because we still support Python 2.7, I believe we should mention both, perhaps like this:

```
we recommend you use venv[link] for virtual environments (or virtualenv[link] for Python 2.7 and 3.4) and pip ...
```

What do you think?

Reply:
Good thought!

Reply:
Updated PR to mention virtualenv for Python  2.7. 

Python 3.4 comes with venv, so I just mentioned 2.7 needing virtualenv. [Source](https://docs.python.org/3/library/venv.htm).